index.rst

```diff
@@ -48,11 +48,302 @@
+By default, the elevation angle used by the elevation dependent component of the LUT is from the onboard inclinometer.
+Only when the elevation angle from the onboard inclinometer is not available would the TMA telemetry be used.
+The source for the elevation angle can be switched between the onboard inclinometer and the TMA telemetry using the
+command ``cmd_selectElevationSource``, which takes one input parameter - 1=Onboard, 2=MTMount.
+The change in the telemetry source triggers the event ``evt_elevationSource``.
+
+######################
+M2 and Camera Hexapods
+######################
+
+The M2 and Camera hexapods have identical control interfaces. The Camera hexapod has ID=1 and M2 hexapod has ID=2.
+
+What the hexapods do is to position themselves to help achieve optimal image quality. So our discussion focuses on how we command the hexapods to the desired positions.
+Here and in the rest of this tech note, position of the hexapod refers to the position (x,y,z) and orientation (dx, dy, dz) relative to the pivot point.
+Unlike for the mirror systems, whose LUTs have to reside with their own control systems due to safety considerations, the LUTs of the hexapods do not have to be with the hexapod CSCs. We choose to have the hexapod LUTs to be part of their owns CSCs for consistency across the AOS.
+
+
+There are various points (x,y,z) and positions (x,y,z,dx,dy,dz) a user needs to understand when operating a hexapod.
+
+- The mechanical zero position, which is (0,0,0,0,0,0) as determined by the actuator encoder readings. When the encoder readings are at their pre-calibrated offset values, the hexapod is at its mechanical zero position.
+- The pivot point, defined as part of the hexapod configuration (right now this is set using SAL command ``cmd_setPivot``; We plan to remove it from SAL). We can define a coordinate system (CS) that is fixed to the hexapod telescope-side base plate. The origin of the CS is at the center of the telescope-side mounting surface, for both the Camera and M2 hexapods. The x, y, and z axes are in parallel with the CCS or M2 CS, when defined using the hexapod hardware (See Sec `5 <https://sitcomtn-003.lsst.io/#m2>`__ and `7 <https://sitcomtn-003.lsst.io/#lsstcam>`__ of SITCOMTN-003). The x, y, and z defined by the pivot configuration is in reference to this CS. For AOS operations, we define the pivot point at L1S1 first vertex and M2 vertex when the hexapod is at their mechanical zero position. Note that the pivot point does not move with the hexapod, i.e., it is fixed relative to the hexapod base plate.
+- Hexapod reported position, as reported by the hexapod control system and CSC. These are in reference to the hexapod CS. The origin of the hexapod CS is at its pivot point. When the system is perfectly aligned, the x, y, and z axes are in parallel with the CCS or M2 CS. When we put the pivot point at L1S1 vertex or M2 vertex, under perfect system alignment, the hexapod CS is the same as the CCS or M2 CS.
```

Comment:
If you think about the x-y plane of the optical coordinate system (see LTS-136), the hexapod base plate is not necessarily parallel to that x-y plane. Neither is the hexapod motion plate. So the fact is that there are several CS involved. What I call the "hexapod CS" is an important CS for the hexapod, because that is the reference used by the hexapod control system to report its position. 

Based on your comment, I now call out the definition of the "hexapod CS" more explicitly. I think that is best way to handle it.

I am actually confused by your comment "the coordinate system that moves when you move the hexapod". This CS doesn't move when you move the hexapod. The CS is fixed relative to the base plate. Its origin, the pivot point, is fixed relative to the base plate (as stated in the text as well). The hexapod moves inside this CS, so that the reported (x,y,z,rx,ry,rz) changes.

Reply:
My worry is that we already have a standard hexapod coordinate system defined in LTS-206 section 2.2. So if you want to define a new one fixed coordinate system, please be very careful that a reader can't confuse it with that one. I was hoping you could avoid the need for a new fixed hexapod coordinate system. There is an obvious movable coordinate system: the one whose origin is at the pivot point and that is simply a translated version of the one in LST-206 when the hexapod is at mechanical zero. And it is that movable system you want to align with the rest of the optics to be in collimation.

Reply:
A few points to make -
- LTS-206 doesn't really define the "Hexapod CS". Fig. 3 is a simple illustration on how to locate the pivot point. It doesn't even tell us the orientation of x and y. But x and y are needed by the setPivot command.
- The CS shown in LTS-206 Fig. 3, with +X, +Y, and +Z marked, is not the "Hexapod CS". Because the rotations reported by the hexapod control system are in reference to P, instead of O.
- The project doesn't really have a proper document that defines all the CS. This has always been a source of confusion. Different components have been using different things. sitcomtn-003 describes how we can (and have to) live with these inconsistencies.

Given the fact that what I wrote confused you, I started trying to add more clarifications to the text. But it was getting longer and longer. So, I've decided to remove that discussion here. It is getting a bit beyond what I wanted to discuss in this document, which is how to command the hexapod. Instead, a new section will be added to sitcomtn-003, where I will define the Hexapod CS.